### ISSUE TITLE
Add options for viewport?

### ISSUE THREAD

**Issue by jprabawa** [rlovtangen/grails-wkhtmltopdf#9]
Viewport option is pretty essential for rendering pages correctly. Can we have this added in the wrapper? :) thanks!

**Comment by rlovtangen** [comment]
You mean this option?

```
 --viewport-size <>              Set viewport size if you have custom
                                      scrollbars or css attribute overflow to
                                      emulate window size
```

Should it be a string value? E.g. "1280x1024"?

**Comment by rlovtangen** [comment]
Fixed by #16
